gwnr/waveform/esigma_utils.py

```diff
@@ -204,13 +209,14 @@ def get_inspiral_esigma_modes(
                                    specific variables. Available orbital variables are:
                                    ['x', 'e', 'l', 'phi', 'phidot', 'r', 'rdot']
         verbose                 -- Verbosity flag
+        return_pycbc_timeseries --
 
     Returns:
     --------
         t                 -- Time grid (in seconds)
         orbital_var_dict  -- Dictionary of evolution of orbital elements.
                              Returned only if "return_orbital_params" is specified.
-        modes_numpy       -- Dictionary of GW modes
+        modes       -- Dictionary of GW modes
```

Comment:
Move "-- Dictionary of GW modes" to the right to align the markers with those above:
```
        modes             -- Dictionary of GW modes
```

Reply:
Done